### PR TITLE
Enable emp dashboard for dynamic orgs

### PR DESCRIPTION
**Pull request by nanditagoclimate**
This is not hardcoded for EUMA but rather for any organization with similar requirement.
The URL is `http://localhost:3000/calculator/start?organization=euma`

1. For a new user
>  When a new user visit this URL, the sign up process is as usual and at the end of account creation they will be linked to EUMA org and the dashboard will be visible in the dropdown.

2. For a non-signed in user
> When a non-signed in user visit this URL, they go through the lifestyle calculator and after completion, they are asked to log in and are then automatically linked to the EUMA dashboard.

3. For a signed in 
> When a signed in user visit this URL, they complete answering lifestyle calculator and then automatically linked to EUMA org.

Please note for case (2) and (3) above: If the user is a part of a different organization, it's over written with EUMA.

Screenshot 2022-05-25 at 6 48 33 PM

Clicking on `EUMA dashboard` leads to `/organization/euma/member-dashboard`
This can also be accessed at `organization/euma/employee-dashboard`
Currently this url rewrite is hardcoded for EUMA. We can come up with a better solution is future if needed.

--------------------------------------------------------------
Pre-deployment step:

- [ ] Create organization for EUMA on production.
- [ ] If  EUMA has a common domain perhaps for directors/founders, that can be added as well so any new/existing users with emails under that domain will automatically be linked to the organization.
- [ ] Key for EUMA should be `euma`

--------------------------------------------------------------
In future to support a new organization:

1. Follow the above pre-deployment steps with the right key and domain
2. URL format will be `/calculator/start?organization=<key>`

### CODE REVIEW

**Review by kallenilver — APPROVED** [review]
Very nice! Good job Nandita!!

I noticed that the region selector was not visible on non-phone screen sizes, so I pushed a commit fixing that I think :)

🥳

**Review by kallenilver — COMMENTED** [review]
detail and green suggesiton, but is it possible for the route to not only be `employee-dashboard` but also `member-dashboard`? Would be nice to have both since `employee-dashboard` is a bit weird for an organisation.

**Comment by nanditagoclimate** [comment]
Didn't quite understand what you mean by this @kallenilver  Could you please explain?

**Review by kallenilver — APPROVED** [review]

File: config/routes.rb [hunk 1]
```diff
@@ -141,6 +141,7 @@
         get 'dashboard/climate-actions', to: 'organization#climate_actions'
         get 'dashboard/projects', to: 'organization#projects'
         get 'employee-dashboard'
+        get :'employee-dashboard', path: 'member-dashboard'
```
[the remark this answers is not in the document]
Comment (nanditagoclimate) [reply config/routes.rb:144]: Yes! can't set this path with 2 strings. Only works with 1 symbol and 1 string.

**Pull request merged**